design/Soft_finality_guarantee.md

```diff
@@ -0,0 +1,157 @@
+# The soft-finality guarantee 
+
+The decentralised POBI protocol links the rollups to the L1 block used as proof, which means that any L1 reorg
+results in an implicit L2 reorg. The finality of a L2 rollup is fully linked to the finality of an L1 block.
+
+During bootstrapping we want a central sequencer to guarantee finality in around ~1second, and for the bride to be decentralised 
+and resistant to reorgs.(
+
+Note: See bridge and finality designs for more details
+
+
+## Problem
+
+The sequencer can only guarantee the ordering of transactions which it receives from the Obscuro users.
```

Comment:
Via other nodes, right? So it's fair to say individual validator nodes may impact ordering. But once they're with the sequencer, they're guaranteed. 

Is there scope to add something the sequencer could use to improve it's understanding of order, like a timestamp?